src/eddymotion/model/base.py

```diff
@@ -27,16 +27,36 @@
 import numpy as np
 from dipy.core.gradients import gradient_table
 from joblib import Parallel, delayed
-
 from eddymotion.exceptions import ModelNotFittedError
 
+#: Minimum value when considering the :math:`S_{0}` DWI signal.
+DEFAULT_MIN_S0 = 1e-5
```

Comment:
Python docstrings can (should) be used to document constants:
```suggestion
DEFAULT_MIN_S0 = 1e-5
"""Minimum value when considering the :math:`S_{0}` DWI signal."""
```